### DISCUSSION TITLE
The operation could not be completed. Unable to locate a Java Runtime

### DISCUSSION THREAD

**Discussion by ivanionut** [sdkman/sdkman-cli#1301] [General]
hi,
i get the following error on SublimeText/macOS:

```
SublimeLinter.plugin.cflint [WARNING]: cflint output:
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.
```

I have already installed java using sdkman:

```
java --version
openjdk 11.0.23 2024-04-16
OpenJDK Runtime Environment Temurin-11.0.23+9 (build 11.0.23+9)
OpenJDK 64-Bit Server VM Temurin-11.0.23+9 (build 11.0.23+9, mixed mode)
```

![java](https://github.com/sdkman/sdkman-cli/assets/2210744/613fd70c-3a18-4f75-a5ba-c5237d6b01c6)

How can I solve this?
Thanks!

**Comment by marc0der** [comment]
You would probably need to start Sublime (or whatever editor you use) from the terminal with your Java version on the path. Alternatively, you could look into the Sublime documentation to see if there is a way to configure a JDK from there.
